problems/removeEvenStrings.js

```diff
@@ -6,8 +6,14 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
-
+function removeEvenStrings(arr) {
+  newArray = [];
+  arr.forEach((el) => {
```

Comment:
The .forEach() array method is not used for side effects. There is a different array method better suited for this function.